.github/workflows/ci.yaml

```
@@ -80,8 +84,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust_version: [ stable, beta, nightly, 1.73.0, "stable minus 1 release", "stable minus 2 releases" ]
-        os: [ ubuntu-latest, windows-latest, macos-latest ]
+        rust_version: [ stable, nightly, 1.73.0 ]
```

I guess clippy is okay if it there only is stable and nightly, since they probably don't remove checks or at least not without a reason

```suggestion
        rust_version: [ stable, nightly ]
```

---

hmm, actually I'm not even sure if clippy has that much of a difference between stable and nightly, since you can enable and disable different categories. So nightly is probably just `clippy::nursery`